Purchases/Misc/Deprecations.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Deprecations.swift
```

Comment:
❤️

Reply:
maybe we should update our Deprecations docs to include this file too? https://github.com/RevenueCat/purchases-ios/blob/main/docs%2FDeprecations.md